dir.props

```diff
@@ -49,6 +46,15 @@
     <ProjectPackagesConfigFile>$(NuGetConfigFile)</ProjectPackagesConfigFile>
   </PropertyGroup>
   
+  <!-- Copy the "_._" file to the output directory, which is used by the NuSpecs to indicate an empty folder -->
+  <ItemGroup>
+    <None Include="$(SourceDir)\nuget\_._">
+      <Link>_._</Link>
```

Comment:
@AndyGerlicher The comment was trying to explain it.  It's used to create an empty folder in a NuGet package, which in this case is used because for .NET Framework there are no libraries in the package itself to reference, they get referenced in the GAC by the frameworkAssemblies element.  See [here](https://github.com/dsplaisted/msbuild/blob/nuget-packages/src/nuget/Microsoft.Build.Framework.nuspec#L36) for how the .nuspec looks.